### ISSUE TITLE
SSL support for postgres datasource connection

### ISSUE THREAD

**Issue by sherfin94** [ToolJet/ToolJet#532]
Right now, connection to postgres databases are made without SSL protection, exposing us to MITM attack. We need an option for the user to upload ssl certificates and connect to postgres using that.

**Comment by sherfin94** [comment]
I'm working on this.

**Comment by sherfin94** [comment]
The above PR ensures that SSL protection is enabled for postgres connection. However, we still need to add support for custom certificates for better protection against MITM attacks.

**Comment by kksd** [comment]
which release this change will be in?

**Comment by kksd** [comment]
is this part of the `v0.6.1` or this will be future releases?  @sherfin94

**Comment by sherfin94** [comment]
Hi @kksd , SSL postgres connection is available in `v.0.6.1`. However, as I mentioned above, support for custom certificates are yet to be added. This will be in future releases.

**Comment by kksd** [comment]
@sherfin94 Thanks for the clarification. That is great to hear. I have `v.0.6.1 ` installed. For my use case, I don't need custom certificates. Could you point me to the doc/reference with instructions on how to enable SSL?

**Comment by komljen** [comment]
Any plans to enable SSL support for MySQL data source as well?

**Comment by sherfin94** [comment]
@komljen It appears that its already enabled. We're considering properly documenting and providing option to enable and disable SSL across data sources.

This is our present connection config with MsSQL, as you can see `encrypted` is set to true:

```const config: Knex.Config = {
      client: 'mssql',
      connection: {
        host : sourceOptions.host,
        user : sourceOptions.username,
        password : sourceOptions.password,
        database : sourceOptions.database,
        port: sourceOptions.port,
        options: {
          encrypt: true
        }
      }
    };

**Comment by komljen** [comment]
I don't see that in the code here https://github.com/ToolJet/ToolJet/blob/main/server/plugins/datasources/mysql/index.ts#L51-L59. Also, with `v0.6.1` installed I had to disable SSL for a particular user to be able to connect.

**Comment by sherfin94** [comment]
Oops, sorry. Yeah, then we need to add the enable/disable option for MySQL. I'm adding this my  to-do list, will get this done as soon as possible. Thank you for pointing out!.

**Comment by kksd** [comment]
Thanks @sherfin94 , Just curious, any update on this

**Comment by sherfin94** [comment]
@kksd Now that you've reminded me, I've started working on it :grimacing: 
I'll miss today's release, but it should be there in the next release.

**Comment by sherfin94** [comment]
@kksd It actually did make to today's release. So its available in `0.7.2`.
